src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/FundingModal/FundingModal.tsx

```diff
@@ -267,10 +267,10 @@ const FundingModal: FC<FundingModalProps> = ({
 
       const motionPayload: ExpenditureFundMotionPayload = {
         ...commonPayload,
-        colony,
-        motionDomainId: Id.RootDomain,
-        fromDomainFundingPotId: Id.RootDomain,
-        fromDomainId: Id.RootDomain,
+        motionDomainId: Id.RootDomain, // @TODO wire this up in the future
```

Comment:
What is the reasoning for wiring this up in the future? Is it because there is no "Created In" field to select in the funding modal?

Reply:
Yeah I've basically started wiring this up and when I tried changing the created in field, I figured out it's missing :D
So I left it as is, and we can decide when to add it in :shrug: but if this PR works, that should be trivial, as we're testing creating motions in all possible scenarios here :crossed_fingers: